App/src/components/Request.js

```diff
@@ -2,37 +2,94 @@
     <li
-    key={request.id}
-    css={{
-      padding: "20px",
-      border: "1px solid black",
-      display: "flex",
-      flexDirection: "column",
-      "&:hover": {
-        cursor: "pointer"
-      }
-    }}
-  >
-    <p>#{request.id}</p>
-    <div>
-      <span>
-        FrontDesk <b>{findName(request.requester_id)}</b> of
-        workshift <b>{Turn[request.current_Shift_id]} </b>
-        want to change workshift with FrontDesk
-      </span>
-      <span>
-        <b> {findName(request.requested_id)} </b>
-        of workshift <b>{Turn[request.requested_Shift_id]} </b>
-      </span>
-    </div>
-    <div css={{ alignSelf: "flex-end" }}>{request.status}</div>
-    <button onClick={() => handleRequestSchedule(request.id)}>
-      See Schedule Request{" "}
-    </button>
-  </li>
+      key={request.id}
+      onClick={onChange}
+      name="text"
+      css={{
+        outline: "none",
+        fontSize: "12px",
+        width: "auto",
+        padding: "10px 20px",
+        minWidth: "119px",
+        cursor: "pointer",
+        backgroundColor: active ? "rgb(224, 232, 249)" : "#fff"
+      }}
+    >
+      <div css={{ display: "flex", flexDirection: "row" }}>
+        <div
+          css={{
+            display: "flex",
+            flexDirection: "column",
+            width: "100px"
+          }}
+        >
+          <p css={{ fontSize: "10px", color: "#98aeeb" }}>Requester</p>
+          <p>{findName(request.requester_id)}</p>
+        </div>
+        <div
+          css={{
+            display: "flex",
+            flexDirection: "column",
+            width: "100px"
+          }}
+        >
+          <p css={{ fontSize: "10px", color: "#98aeeb" }}>From</p>
+          <p>{Turn[request.current_Shift_id]}</p>
+        </div>
+        <div
+          css={{
+            display: "flex",
+            flexDirection: "column",
+            width: "100px"
+          }}
+        >
+          <p css={{ fontSize: "10px", color: "#98aeeb" }}>Status</p>
+          <p>{request.status}</p>
+        </div>
+      </div>
+      <div css={{ display: "flex", flexDirection: "row" }}>
+        <div
+          css={{
+            display: "flex",
+            flexDirection: "column",
+            width: "100px"
+          }}
+        >
+          <p css={{ fontSize: "10px", color: "#98aeeb" }}>Requested</p>
+          <p>{findName(request.requested_id)}</p>
+        </div>
+        <div
+          css={{
+            display: "flex",
+            flexDirection: "column",
+            width: "100px"
+          }}
+        >
+          <p css={{ fontSize: "10px", color: "#98aeeb" }}>To</p>
+          <p>{Turn[request.requested_Shift_id]}</p>
+        </div>
+        <div
+          css={{
+            display: "flex",
+            flexDirection: "column",
+            width: "100px"
+          }}
+        >
+          <p css={{ fontSize: "10px", color: "#98aeeb" }}>On</p>
+          <p>7/29/2019</p>
+        </div>
+      </div>
+    </li>
```

Comment:
If you change the li to an article remember to change it here too.
```suggestion
    </article>
```